.github/workflows/pr.yaml

```diff
@@ -2,17 +2,7 @@ name: pr
 
 on: [pull_request]
 
-permissions:
-  actions: read
```

Comment:
just to make sure, we don't need this either? I assume so, as our workflows seem to be isolated from each other, but wanted to confirm anyway

Reply:
The checks on this PR ran without the permissions, seems to work fine.